Packs/AWS-SQS/Integrations/AWS-SQS/AWS-SQS.py

```diff
@@ -238,7 +238,7 @@ def main():
     timeout = params.get('timeout')
     retries = params.get('retries') or 5
     aws_queue_url = params.get('queueUrl')
-    max_fetch = min(arg_to_number(params.get('max_fetch', 10)), 100)
+    max_fetch = min(arg_to_number(params.get('max_fetch', 10)) or 10, 100)
```

Comment:
@MosheEichler Why do you need the or 10 ? if max_fetch is None it will return 10 anyway and if it isn't a valid number it will raise an exception

Reply:
@gal-forer Because of this failure
https://app.circleci.com/pipelines/github/demisto/content/188348/workflows/35d0a091-9c4c-489f-a971-9e4123e6738c/jobs/563982?invite=true#step-107-172
So either we ignore it or we make this change...